manticore/ethereum.py

```diff
@@ -631,6 +631,26 @@ def did_evm_write_storage_callback(self, state, address, offset, value):
         state.context.setdefault('{:s}.initialized_storage'.format(self.name), set()).add((address, offset))
 
 
+class DetectMultipleSends(Detector):
+    '''
+    Detects multiple sends to the same address within a transaction
```

Comment:
I actually don't think that it's particularly a problem if the transactions are to the same address. My understanding is that it's just a general not-great practice to have more than one transaction initiated (.send, .transfer, .call, ...) within a function